README.adoc

```diff
@@ -88,6 +88,22 @@ assert_equal(true, actual)
 assert(actual)
 ----
 
+=== Refute false [[refute-false]]
```

Comment:
We should update this to `Refute Falsey` to maintain consistency.

Reply:
@abhaynikam, @koic made a good observation on `refute falsey`, check this: https://github.com/rubocop-hq/minitest-style-guide/pull/10#discussion_r314552075